paddle/platform/profiler.cc

```diff
@@ -13,12 +13,18 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 #include "paddle/platform/profiler.h"
+#include <iomanip>
+#include <map>
+#include "gflags/gflags.h"
```

Comment:
The `gflags` is not used. This header can be removed.

Reply:
Done